HelpSource/Guides/MultiClient_Setups.schelp

```diff
@@ -0,0 +1,176 @@
+TITLE:: MultiClient_Setups
+SUMMARY:: How to handle clientIDs and id allocation in multi-client/server setups
+CATEGORIES:: Guides
+related:: Classes/ServerOptions, Classes/Server
+KEYWORD:: server, multi-client
+
+section:: Multi client-server setups - discussion and tests
+
+OSC communication between SC and its sound server offers many options for network music: Multiple computers can run both supercollider and associated sound servers.
+For clarity, the "server program" refers a running scsynth or supernova process. The "server object" AKA "client" is the server's representation in sclang, such as code::Server.local, s, Server(\elsewhere, NetAddr("123.234.56.78"))::.
```

Comment:
do you mean "refers to" ?
also, maybe call it "server process" rather than "program".